<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScatterWidget</class>
 <widget class="QWidget" name="ScatterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>823</width>
    <height>752</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scatter Plot</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="margin">
    <number>5</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="handleWidth">
      <number>7</number>
     </property>
     <widget class="MplWidget" name="mplWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>300</height>
       </size>
      </property>
      <property name="toolTip">
       <string/>
      </property>
     </widget>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="axisLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetMaximumSize</enum>
        </property>
        <item>
         <layout class="QHBoxLayout" name="xAxisLayout">
          <item>
           <widget class="QLabel" name="xlabel">
            <property name="text">
             <string>x axis</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="xAxisComboBox">
            <property name="toolTip">
             <string>Set which attribute is plotted on the x axis</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="xLogCheckBox">
            <property name="toolTip">
             <string>Toggle on/off log scaling on the x axis</string>
            </property>
            <property name="text">
             <string>log</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="xFlipCheckBox">
            <property name="toolTip">
             <string>Flip/unflip the order of the x axis</string>
            </property>
            <property name="text">
             <string>flip</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="yAxisLayout">
          <item>
           <widget class="QLabel" name="ylabel">
            <property name="text">
             <string>y axis</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="yAxisComboBox">
            <property name="toolTip">
             <string>Set which attribute is plotted on the y axis</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="yLogCheckBox">
            <property name="toolTip">
             <string>Toggle on/off log scaling on the y axis</string>
            </property>
            <property name="text">
             <string>log</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="yFlipCheckBox">
            <property name="toolTip">
             <string>Flip/unflip the order of the y axis</string>
            </property>
            <property name="text">
             <string>flip</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTreeWidget" name="layerTree">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>100</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <column>
           <property name="text">
            <string>Layer</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Color</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Symbol</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Size</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="layerAddButton">
            <property name="toolTip">
             <string>Add a new subset or data set</string>
            </property>
            <property name="statusTip">
             <string/>
            </property>
            <property name="whatsThis">
             <string/>
            </property>
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="layerRemoveButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Remove a subset or data set from all visualizations</string>
            </property>
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="linkButton">
            <property name="toolTip">
             <string>Link subsets</string>
            </property>
            <property name="text">
             <string>Link</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header>mplwidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>xAxisComboBox</tabstop>
  <tabstop>xLogCheckBox</tabstop>
  <tabstop>xFlipCheckBox</tabstop>
  <tabstop>yAxisComboBox</tabstop>
  <tabstop>yLogCheckBox</tabstop>
  <tabstop>yFlipCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
